nxdrive/data/i18n/i18n.json

```
@@ -132,6 +133,10 @@
     "FATAL_ERROR_TITLE": "%1 - Fatal error",
     "FATAL_ERROR_UPDATE_BTN": "Download latest version",
     "FATAL_ERROR_UPDATE_TOOLTIP": "We advise you to update to the latest version of %1",
+    "FEATURE_AUTO_UPDATE" : "Enable or disable the update mechanism of Nuxeo Drive.",
```

```suggestion
    "FEATURE_AUTO_UPDATE" : "Enable or disable the update mechanism of %1",
```
And pass the `APP_NAME` constant to the translator.